packages/atomic/src/components/facets-v1/facet-number-input/facet-number-input.tsx

```diff
@@ -31,44 +45,70 @@ export class FacetNumberInput {
       return;
     }
 
-    this.start = this.filterState.range?.start;
-    this.end = this.filterState.range?.end;
+    this.start = this.filterState.range?.start.toString() || '';
+    this.end = this.filterState.range?.end.toString() || '';
   }
 
   private apply() {
-    this.onApply && this.onApply();
+    this.applyInput.emit();
+    this.filter.setRange({
+      start: parseFloat(this.start),
+      end: parseFloat(this.end),
+    });
+  }
 
-    try {
-      this.filter.setRange({
-        start: this.start!,
-        end: this.end!,
-      });
-      this.isRangeInvalid = false;
-    } catch (error) {
-      this.bindings.engine.logger.error(error);
-      this.isRangeInvalid = true;
+  private validateInput(value: string) {
```

Comment:
What is the format of the `value` here ? 

Is it really any string ? 
If you wanted to break this function, what would a end user need to input ?

Are we leveraging all the standard browser function for `type=number` ?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number

For example, we have access to `min`, `max`, `step` (for integer or even for decimal), `pattern` for validation, etc... I would assume using as much of those attributes as possible means screen reader will be able to benefit from it also.


Also, it does not look like a validation function to me, more like a `sanitize` or `trim` ?